index.html

```diff
@@ -114,77 +114,119 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
-	 
-	      <!-- ________ *Cesar959 card START ________  -->
-	      <div class="card">
-		      <p class="name">Cesar dos Santos de Almeida</p>
-		      <p class="contact">
-		        <a href="https://github.com/Cesar959" target="_blank"><i class="fab fa-github"></i></a>
-		        <a href="https://www.linkedin.com/in/cesarsantosalmeida/" target="_blank"><i class="fab fa-linkedin"></i></a>
-		      </p>
-		      <p class="about">Sou programador júnior, Apaixonado por programação web, formado no curso técnico em informatica do Senac de taubaté.</p>
-		      <div class="resources">
-		      <p>3 Useful Dev Resources</p>
-		      <ul>
-			      <li>
-			        <a href="https://readme.so/" target="_blank" title="Readme.so">
-			          Readme.so
-			        </a>
-			      </li>
-			      <li>
-			        <a href="https://www.frontendmentor.io/challenges" target="_blank" title="Frontend Mentor">
-			          Frontend Mentor
-			        </a>
-			      </li>
-			      <li>
-			        <a href="https://www.dailyui.co/" target="_blank" title="Daily UI">
-			          Daily UI
-			        </a>
-			      </li>
-		      </ul>
-		    </div>
-	    </div>
-	    <!-- ________*Cesar959 card END ________  -->
-        
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Kris</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/refaktor_dev" target="_blank">refaktor_dev</a>
+          </p>
+          <p class="about">Python developer</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://uptimerobot.com/" target="_blank" title="uptimerobot.com">
+                  uptimerobot.com
+                </a>
+              </li>
+              <li>
+                <a href="https://docs.djangoproject.com/en/4.0/contents/" target="_blank" title="djangoproject.com">
+                  djangoproject.com
+                </a>
+              </li>
+              <li>
+                <a href="https://python.swaroopch.com/" target="_blank" title="A Byte of Python">
+                  A Byte of Python
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change (add your name) this comment as well. 
Otherwise everyone will have a hard time identifying your code.